oca/tests/test_api_foreach.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import unittest
+
+from .. import api
```

Comment:
Bah you had to mention this, didn't you?! I fully agree, but I'm also completely at a loss regarding what to replace it with. I settled on forcing `@oca.api.foreach()` for usage, but I don't think that's very intuitive - especially after you noticing too.

What if we maybe renamed the concept of API to something a bit more specific? `oca.model.foreach`? `oca.record.foreach`?